config/deploy.rb

```diff
@@ -0,0 +1,79 @@
+# config valid only for Capistrano 3.1
+lock '3.9'
+
+############################################
+# Setup project
+############################################
+
+set :application, "chappress"
+set :repo_url, "git@github.com:chapmanu/chap-press.git"
+
+set :git_strategy, SubmoduleStrategy
```

Comment:
Yes from Blogs. Refering to `SubmodulesStrategy`? I think I can remove it. I just keep it since it was from the blogs.